e2e/cypress/integration/02-apd/01-apd-basics.cy.js

```diff
@@ -253,6 +253,68 @@ describe('APD Basics', { tags: ['@apd', '@default'] }, () => {
     });
   });
 
+  describe('tests admin check validation errors', () => {
+    it('tests non-subform validation errors', () => {
+      cy.contains('Export and Submit').click();
```

Comment:
it's probably good to make this a command so it's easier to turn it on and off, I imagine we'll be doing that a lot soon